src/internal/operators/switchMap.ts

```diff
@@ -9,11 +10,19 @@ import { lift } from '../util/lift';
 import { SimpleInnerSubscriber, innerSubscribe, SimpleOuterSubscriber } from '../innerSubscribe';
 
 /* tslint:disable:max-line-length */
```

Comment:
I think this PR has addressed the max-length now, so the tslint max-line can be removed.